app/helpers/payload_parser.rb

```diff
@@ -0,0 +1,5 @@
+module PayloadParser
```

Comment:
`app/helpers` directory has another use in rails, it's actually view helpers

Reply:
`app/services/concerns` is ok?

Reply:
hmmm concerns has a particular meaning for rails (which I dislike) somethine like `app/lib`?